### ISSUE TITLE
Functionality from Eternal Script

### ISSUE THREAD

**Issue by HalinSky** [Maran23/script-ide#58]
Unfortunatedly it appears godot addons interact poorly and this limits what can you get in terms of window modifications.

This addon is very useful but is not compatible with script-ide: 
https://raw.githubusercontent.com/masoud-ata/godot-eternal-scripts/master/assets/before_after_animation.gif

Mainly the "script shortcut" from scene would be amazing.

**Comment by zegenie** [comment]
Yes, it seems like making the docks appear in a customized location instead of fixed to the editor side will solve compatibility with other plugins, but more importantly it makes it easier to use for developers with customized Godot editors / interfaces :smile: 

Thanks for making the plugin!

**Comment by Maran23** [comment]
Closing in favor of https://github.com/Maran23/script-ide/issues/60
